docs/src/testing/testing.md

```diff
@@ -72,3 +72,25 @@ Failure data:
 
 Tests: 0 passed, 1 failed, 0 skipped
 ```
+
+## Expected failures
+
+Sometimes you want to mark a test as expected to fail. This is useful when you are working on a feature that is not yet, or when you want to verify that an action fails as expected.
+
+To mark a test as expected to fail, use the `#[should_panic]` attribute. You can pass the expected failure message as an argument to the attribute to verify that the test fails with the expected message with `#[should_panic(expected: ('panic message', 'eventual second message',))]`.
+
+```text
```

Comment:
```suggestion
```rust
```

Reply:
I can't use rust formatting because of the single quotes here. It makes the output super weird